File: 프로그래머스/0/120895. 인덱스 바꾸기/인덱스 바꾸기.java
class Solution {
    public String solution(String my_string, int num1, int num2) {
        String[] s_list = my_string.split("");
        String answer = "";
        
        String n1 = s_list[num1];
        String n2 = s_list[num2];
        s_list[num1]= n2;
        s_list[num2]= n1;
        
        return String.join("", s_list);
    }
}

File: 프로그래머스/0/120895. 인덱스 바꾸기/인덱스 바꾸기.py
def solution(my_string, num1, num2):
    answer = ''
    s_list = list(my_string)
    n1 = s_list[num1]
    n2 = s_list[num2]
    s_list[num1] = n2
    s_list[num2] = n1
    

    return "".join(s_list)